examples/webgl_loader_x.html

```diff
@@ -163,7 +163,7 @@
 
             //download Model file
 
-            loader.load( [ 'models/xfile/SSR06_model.x', { zflag: false } ], function ( object ) {
+            loader.load( [ 'models/xfile/SSR06_model.x' ], function ( object ) {
```

Comment:
There are additional uses of `zFlag` in this file.

Reply:
The second argument is made so that it can be omitted even if omitted.

If [zflag] is omitted, it behaves like [zflag = false].
If there was a person using [zflag= true],  I recommend that [scale.x = -1] be set after processing with the loader.

Reply:
You should remove `zflag` entirely.
```javascript
if ( this.options.zflag ) {
...
loader2.load( [ 'models/xfile/' + animeName + '.x', { zflag: false, putPos: false, putScl: false } ] . . .
```

Reply:
For the 4th time, please remove all occurrences of `zflag`.
```javascript
if ( this.options.zflag ) {
```

Reply:
done